charts/victoria-metrics-auth/templates/clusterrolebinding.yaml

```diff
@@ -0,0 +1,24 @@
+{{- if .Values.rbac.create -}}
```

Comment:
vmauth do not needed this role.

Reply:
Thanks @f41gh7, I will remove this file in the next commit.